app/models/incomeSourceDetails/viewmodels/ManageIncomeSourceDetailsViewModel.scala

```diff
@@ -28,7 +28,8 @@ case class ManageIncomeSourceDetailsViewModel(incomeSourceId: IncomeSourceId,
                                               tradingStartDate: Option[LocalDate],
                                               address: Option[AddressModel],
                                               isTraditionalAccountingMethod: Boolean,
-                                              itsaHasMandatedOrVoluntaryStatusCurrentYear: Boolean,
+                                              latencyYearTwoIsQuarterly: Option[Boolean],
+                                              latencyYearOneIsQuarterly: Option[Boolean],
                                               taxYearOneCrystallised: Option[Boolean],
                                               taxYearTwoCrystallised: Option[Boolean],
```

Comment:
I've noticed latency years and crystallisation years both have year1 and year2 which seem to be coupled throughout, would it be worth creating case classes for these?